src/components/templates/Playa/Playa.tsx

```diff
@@ -394,29 +396,22 @@ const Playa = () => {
   }, [camp, venues, showVenue]);
 
   const [openVenues, setOpenVenues] = useState<OnlineStatsData["openVenues"]>();
-  useEffect(() => {
-    const getOnlineStats = firebase
+
+  useInterval(() => {
+    firebase
       .functions()
-      .httpsCallable("stats-getLiveAndFutureEvents");
-    const updateStats = () => {
-      getOnlineStats()
-        .then((result) => {
-          const { openVenues } = result.data as OnlineStatsData;
-          setOpenVenues(
-            profile?.kidsMode
-              ? openVenues.filter((v) => !v.venue.adultContent)
-              : openVenues
-          );
-          //setOpenVenues(openVenues);
-        })
-        .catch(() => {}); // REVISIT: consider a bug report tool
-    };
-    updateStats();
-    const id = setInterval(() => {
-      updateStats();
-    }, REFETCH_SCHEDULE_MS);
-    return () => clearInterval(id);
-  }, [profile]);
+      .httpsCallable("stats-getLiveAndFutureEvents")()
+      .then((result) => {
+        const { openVenues } = result.data as OnlineStatsData;
+        setOpenVenues(
+          profile?.kidsMode
+            ? openVenues.filter((v) => !v.venue.adultContent)
+            : openVenues
+        );
+        //setOpenVenues(openVenues);
```

Comment:
nit: commented code

Reply:
This was existing in the template prior, so I left it as was. Either way this whole thing is 🔥  once https://github.com/sparkletown/sparkle/pull/833 is ready